src/styl/site.styl

```diff
@@ -60,6 +71,15 @@ body
 .line-larger
   line-height: $font-leading-larger
 
+.font-logo
+  font-family: $font-logo
+
+.font-ruberta
```

Comment:
is this font only ever used for the text that says Ruberta? Are they any other contexts for it? If not perhaps we could do a extreme subset down to just those letters, or at least we should do some subsetting.

Reply:
idk what you mean by subsetting but i'll make a task